### DISCUSSION TITLE
Test video conferencing

### DISCUSSION THREAD

**Discussion by Vignesh-E** [baresip/baresip#1924] [Q&A]
need to test Video conferencing using baresip.
I want to know configuration to do conferencing, I tested by enabling mixminus module, but it is  not working properly.

Eg:
s-server
c1-client1
c2-client2
s getting both c1 and c2 frames
c1 getting s frames
c2 gets nothing
 Is this the actual scenario?

**Comment by sreimers** [comment]
`mixminus` is handling audio only. For a side project I have a baresip vidmix module (uses the rem vidmix api) that could handle that, but needs some modifications for a generic use case. if there is interest i can customize this after my vacation and create a PR.

**Reply by Vignesh-E** [reply]
Thanks @sreimers 
a. Earlier I have tested with baresip, server was able to receive all clients video and all clients able to receive server video. Tested this scenario and it worked. Now it is not working, is there anything baresip has changed now?.
b. Can I send video from server to two clients?

**Reply by sreimers** [reply]
There is a new config option `call_hold_other_calls` by default its `yes` and other calls are on hold. You can disable this with `no`. Maybe this affects you.

**Reply by GOBIKRISHNAN-R** [reply]
I'm also facing the same issue. I tried the config option(call_hold_other_calls) but there's no difference. If I'm the baresip server, Can I send my video to two different clients?

**Reply by Vignesh-E** [reply]
Hi @sreimers Can you create a PR for vidmix?

**Reply by bala-t** [reply]
> `mixminus` is handling audio only. For a side project I have a baresip vidmix module (uses the rem vidmix api) that could handle that, but needs some modifications for a generic use case. if there is interest i can customize this after my vacation and create a PR.

Is there any update in this??

**Reply by sreimers** [reply]
It's on the roadmap: https://github.com/baresip/baresip/wiki/Roadmap#video

If I have enough spare time, I will take the task.
